### ISSUE TITLE
Glyph position of U+11C1 ᇁ

### ISSUE THREAD

**Issue by acuteaccent** [adobe-fonts/source-han-sans#90]
![u11c1-shs](https://cloud.githubusercontent.com/assets/8185770/6766895/282bcafa-cfd6-11e4-847a-33f8decaf0d4.png)

When U+11C1 ᇁ is directly followed by a vowel that goes under the leading consonant (U+1169 ᅩ, U+116E ᅮ, U+1173 ᅳ, U+119E ᆞ, etc.), the glyph position of U+11C1 ᇁ looks a bit awkward. I think it needs to be moved to the right a little bit, so that it is centered.

Comparing positions with U+11AF ᆯ, without lines:
![rieul-phieuph](https://cloud.githubusercontent.com/assets/8185770/6766922/608c01de-cfd7-11e4-870c-9dc6fcc2136c.png)
Comparing positions with U+11AF ᆯ, with lines:
![rieul-phieuph-withlines](https://cloud.githubusercontent.com/assets/8185770/6766923/6827360c-cfd7-11e4-95af-a82f7e4b56da.png)

Sorry for submitting an issue when the new version is about to be released.

**Comment by kenlunde** [comment]
When I checked the combining glyphs for U+11C1 with those of others, the one for uni11C1.tjmo04 (CID+65033) seems to be shifted more to the left than other uni*.tjmo04 glyphs. The shifting amount seems to match your observations.

![uni11c1-error](https://cloud.githubusercontent.com/assets/3988788/6766939/268f8306-cfd8-11e4-80c0-01859e87c1a8.png)

**Comment by kenlunde** [comment]
I looked more deeply at this issue, and will be adding the following to the "Known Issues" section of the Version 1.002 ReadMe PDF: "The glyphs for *uni11C1.tjmo04* (CID+65033) and *uni11D6.tjmo04* (CID+65054) need to be shifted to the right in order to better align with other jamo when combined. This issue is manifested in the Heavy weight, but affects all other weights except for ExtraLight due to interpolation."

In other words, if you use the ExtraLight weight, you shouldn't see this issue, but if you use the Heavy weight, it should look even worse. Note that I detected another glyph that should exhibit this issue: *uni11D6.tjmo04* (CID+65054).

**Comment by acuteaccent** [comment]
I see.
![11c1and11d6](https://cloud.githubusercontent.com/assets/8185770/6767441/2dee2e4e-cfed-11e4-8307-fd7540148ac1.png)

**Comment by kenlunde** [comment]
Consolidated with Issue #99.

**Comment by kenlunde** [comment]
Consolidated with Issue #178.
